TombEngine/Resources.rc

```diff
@@ -27,8 +17,8 @@ LANGUAGE LANG_ENGLISH, SUBLANG_NEUTRAL
 //
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION TEN_MAJOR_VERSION,TEN_MINOR_VERSION,TEN_BUILD_NUMBER,TEN_REVISION_NUMBER
- PRODUCTVERSION TEN_MAJOR_VERSION, TEN_MINOR_VERSION, TEN_BUILD_NUMBER, TEN_REVISION_NUMBER
+ FILEVERSION 1,10,0,2
```

Comment:
Is version now hardcoded, or this file is auto-recreated by some script? `TEN_MAJOR_VERSION` etc macros were automatically upgrading version number when `Version.h` was changed.